frontend/device/android/device.lua

```diff
@@ -30,9 +30,9 @@ function Device:init()
             logger.dbg("Android application event", ev.code)
             if ev.code == C.APP_CMD_SAVE_STATE then
                 return "SaveState"
-            elseif ev.code == C.APP_CMD_GAINED_FOCUS then
-                this.device.screen:refreshFull()
-            elseif ev.code == C.APP_CMD_WINDOW_REDRAW_NEEDED then
+            elseif ev.code == C.APP_CMD_GAINED_FOCUS
+                or ev.code == C.APP_CMD_INIT_WINDOW
```

Comment:
I wonder if this might also fix the other issue with the brightness level.

Reply:
Hopefully :+1: